docs/DeveloperGuide.md

```diff
@@ -1448,15 +1458,19 @@ testers are expected to do more *exploratory* testing.
 
    1. Test case: Minimal information e.g. `add ct/w n/John Ptd Ltd p/98766789 addr/John street, block 123
       , #01-01`<br>
-      Expected: Adds a warehouse with the above details to the list and displayed on the GUI.
-   1. Test case: With remarks e.g. `add ct/w n/John Ptd Ltd p/98766789 addr/John street, block 123, #01-01
-      r/Largest warehouse`<br>
-      Expected: Adds the warehouse to the list, including the remark
+      Expected: Adds a warehouse with the above details to the warehouse list and is displayed on the GUI.
+   1. Test case: Duplicate warehouse by name e.g. `add ct/w n/John Ptd Ltd p/99999999 addr/John 2 street
+      , block 222, #02-02`<br>
+      Expected: No warehouse is added. Adds the warehouse to the list, including the remark
```

Comment:
Is there a mistake here? "No warehouse is added. Adds the warehouse to the list"

Reply:
Yup, i was in the midst of editing both and got distacted by my UML hahahaa